win32/src/winapi/kernel32/sync.rs

```diff
@@ -1,30 +1,48 @@
 //! Synchronization.  Currently all no-ops as we don't support threads.
 
-use crate::{winapi::types::HANDLE, Machine};
+use crate::{winapi::types::HEVENT, Machine};
 
 const TRACE_CONTEXT: &'static str = "kernel32/misc";
 
+pub struct EventHandle {
+    state: bool,
+}
+
 #[win32_derive::dllexport]
 pub fn WaitForSingleObject(
     _machine: &mut Machine,
-    hHandle: HANDLE<()>,
+    hHandle: HEVENT,
     dwMilliseconds: u32,
 ) -> u32 {
     todo!()
 }
 
 #[win32_derive::dllexport]
 pub fn CreateEventA(
-    _machine: &mut Machine,
+    machine: &mut Machine,
     lpEventAttributes: u32,
     bManualReset: bool,
     bInitialState: bool,
     lpName: Option<&str>,
-) -> HANDLE<()> {
-    todo!()
+) -> HEVENT {
+    if lpName.is_some() {
+        log::warn!("CreateEventA: named events not supported");
+        todo!();
```

Comment:
You can pass the string to todo like

todo!("CreateEventA: named events not supported");